documentation/configuration.md

```diff
@@ -21,5 +21,19 @@ metrics: {
             enabled: true
         }
     }
+
+    jolokia {
+      # Enable or disable the Jolokia-based metrics/v2 endpoint.
+      # Default is true.
+      enabled: false
+
+      # Configure any of the settings listed at:
+      #   https://jolokia.org/reference/html/agents.html#war-agent-installation
+      config: {
+        # Specify a custom security policy:
+        #  https://jolokia.org/reference/html/security.html
+        policyLocation: "file:///etc/puppetlabs/<service name>/jolokia-access.xml"
```

Comment:
As for renaming the section to `servlet-params`, these settings are named for entries in the `org.jolokia.config.ConfigKey` enum and are used to construct a `org.jolokia.config.Configuration` object. Due to the current implementation, that does happen as the result of passing servlet parameters, but I feel like that's more of an implementation detail.

Put another way, if we decided in the future to refactor and go with the approach of wrapping the Jolokia HTTP handler classes in Clojure, as done in PR #33, the structure and allowable values of the `config` map wouldn't change, even though the data would no longer be passed as servlet parameters. Additionally, there may be generic servlet settings that can't be put in this block as they won't pass schema validation because they aren't listed in the `org.jolokia.config.ConfigKey` enum.